components/pipeline-service/staging/base/main-pipeline-service-configuration.yaml

```diff
@@ -886,6 +886,19 @@ metadata:
   namespace: tekton-results
 ---
 apiVersion: v1
+kind: ConfigMap
```

Comment:
yeah this configmap def should be removed once you mane the TektonConfig changes and run deploy-config.sh like I just described.

Reply:
Done, thank you!